queries/just/highlights.scm

```diff
@@ -0,0 +1,39 @@
+
+(assignment (NAME) @variable)
+(alias (NAME) @variable)
+(value (NAME) @variable)
+(parameter (NAME) @variable)
+(setting (NAME) @keyword)
+(setting "shell" @keyword)
+
+(call (NAME) @function)
+(dependency (NAME) @function)
+(depcall (NAME) @function)
+(recipeheader (NAME) @function)
+
+(depcall (expression) @parameter)
+(parameter) @parameter
+(variadic_parameters) @parameter
+
+["if" "else"] @conditional
+
+(string) @string
+
+(boolean ["true" "false"]) @boolean
+
+(comment) @comment
+
+(recipe
+  (body) @text.literal.block (#set! "priority" 90))
+
+(shebang_recipe
+  (shebang interpreter: (TEXT) @keyword) @comment
+  (shebang_body) @text.literal.block (#set! "priority" 90))
```

Comment:
Is this a deliberate priority reducing for bash/python3? or what

Reply:
Because if it's only to be injected, it's better to just not have any highlights applied to the node at all

Reply:
@lucario387 It's intended to be overridden by injected syntax highlighting, yes. I followed the instructions here:

https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#text

It works as a nice fallback if the user picks a language that doesn't have an installed parser.

Reply:
I don't see why it's a necessity to highlight raw text.